### PR TITLE
Unified naming for few services from "items" package

### PR DESCRIPTION
**Pull request by smadbe**
Handling part of #536

### CODE REVIEW

File: app/api/items/list_attempts.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
-// swagger:model itemAttemptsViewResponseRow
-type itemAttemptsViewResponseRow struct {
+// swagger:model attemptListResponseRow
+type attemptListResponseRow struct {
```
Comment (zenovich) [inline app/api/items/list_attempts.go:14]: attemptsListResponseRow?

File: app/api/items/list_attempts.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ type itemAttemptsViewResponseRow struct {
 	} `json:"user_creator" gorm:"embedded;embedded_prefix:user_creator__"`
 }
 
-// swagger:operation GET /items/{item_id}/attempts items itemAttemptsView
+// swagger:operation GET /items/{item_id}/attempts items attemptList
```
Comment (zenovich) [inline app/api/items/list_attempts.go:38]: attemptsList?

File: app/api/items/list_attempts.go [hunk 3]
```diff
@@ -82,7 +82,7 @@ type itemAttemptsViewResponseRow struct {
 //     schema:
 //       type: array
 //       items:
-//         "$ref": "#/definitions/itemAttemptsViewResponseRow"
+//         "$ref": "#/definitions/attemptListResponseRow"
```
Comment (zenovich) [inline app/api/items/list_attempts.go:85]: attemptsListResponseRow?

Comment (smadbe) [reply app/api/items/list_attempts.go]: fixed

File: app/api/items/list_attempts.go (hunk 2 above)
Comment (smadbe) [reply app/api/items/list_attempts.go]: fixed

File: app/api/items/list_attempts.go (hunk 1 above)
Comment (smadbe) [reply app/api/items/list_attempts.go]: fixed

File: app/api/items/get_breadcrumbs.go [hunk 4]
```diff
@@ -11,12 +11,12 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
-// swagger:operation GET /items/{ids}/breadcrumbs items itemsBreadCrumbsData
+// swagger:operation GET /items/{ids}/breadcrumbs items itemBreadcrumbGet
```
[the remark this answers is not in the document]
Comment (smadbe) [reply app/api/items/get_breadcrumbs.go]: fixed

**Comment by smadbe** [comment]
Ok, that's all for this PR (I'll not add new services in it). To be merged as soon as the review is ok.

**Pull request merged**